lib/EnsEMBL/REST/Controller/Info.pm

```diff
@@ -155,6 +155,65 @@ sub biotypes_GET :Local :Args(1) {
   return;
 }
 
+sub biotype_groups_GET { }
+
+sub biotype_groups : Path("biotypes/groups") Args(0) ActionClass('REST') {
+  my ($self, $c) = @_;
+
+  my $groups;
+
+  try {
+    $groups = $c->model('Biotype')->fetch_biotype_groups;
+
+  } catch {
+    $c->go('ReturnError', 'from_ensembl', [qq{$_}]) if $_ =~ /STACK/;
+    $c->go('ReturnError', 'custom', [qq{$_}]);
+  };
+
+  $self->status_ok($c, entity => $groups);
+  return;
+}
+
+sub biotype_group_GET { }
+
+sub biotype_group : Path("biotypes/groups") Args(1) ActionClass('REST') {
+  my ($self, $c, $group) = @_;
+
+  my $biotypes;
+
+  try {
+    $biotypes = $c->model('Biotype')->fetch_biotypes_by_group($group);
+
+  } catch {
+    $c->go('ReturnError', 'from_ensembl', [qq{$_}]) if $_ =~ /STACK/;
+    $c->go('ReturnError', 'custom', [qq{$_}]);
+  };
+
+  $self->status_ok($c, entity => $biotypes);
+  return;
+}
+
+sub biotype_name_GET { }
+
+sub biotype_name : Path("biotypes/name") CaptureArgs(1) ActionClass('REST') {
+  my ($self, $c, $name) = @_;
+
+  $c->go('ReturnError', 'custom', ["Missing argument ':name' for endpoint info/biotypes/name/:name"]) unless $name;
+
+  my $biotypes;
+
+  try {
+    $biotypes = $c->model('Biotype')->fetch_biotypes_by_name($name);
```

Comment:
This silently fails when a request is given for a biotype that doesn't exist. An empty listref is the response. Normally it would be more conventional to get a 404 with an explanatory message.